README.md

```diff
@@ -1,376 +1,242 @@
-# Spectrum CSS
+<div align="right">
```

Comment:
```suggestion
<div align="right">
```
Looks like we might have lost a closing div on this one in the rebase, which is throwing off the alignment on the bullet points near the top.